app/views/return-requirements/check.njk

```diff
@@ -110,202 +117,204 @@
-                    items: [
-                      {
-                        href: "frequency-reported/" + requirement.index ,
-                        text: "Change",
-                        visuallyHiddenText: "Reporting"
-                      }
-                    ]
+                ]
+              }
+              },
+              {
+                key: {
+                text: "Reporting"
+              },
+                value: {
+                html: requirement.frequencyReported | capitalize
+              },
+                actions: {
+                items: [
+                  {
+                    href: "frequency-reported/" + requirement.index ,
+                    text: "Change",
+                    visuallyHiddenText: "Reporting"
                   }
-                },
-                {
-                  key: {
-                    text: "Agreements exceptions"
-                  },
-                  value: {
-                    html: "requirement.agreementsExceptions"
-                  },
-                  actions: {
-                    items: [
-                      {
-                        href: "agreements-exceptions/" + requirement.index ,
-                        text: "Change",
-                        visuallyHiddenText: "Agreements exception"
-                      }
-                    ]
+                ]
+              }
+              },
+              {
+                key: {
+                text: "Agreements exceptions"
+              },
+                value: {
+                html: "requirement.agreementsExceptions"
+              },
+                actions: {
+                items: [
+                  {
+                    href: "agreements-exceptions/" + requirement.index ,
+                    text: "Change",
+                    visuallyHiddenText: "Agreements exception"
                   }
-                }
-              ]
-            }) }}
-          {% if requirements.length >= 2 %}
-            {{ govukButton({
-              text: "Remove requirement",
-              classes: "govuk-button--warning",
-              preventDoubleClick: true,
-              href: "/system/return-requirements/" + sessionId + "/remove/" + requirement.index
-            }) }}
-          {% endif %}
+                ]
+              }
+              },
+              {
+                key: {
+                html:  govukButton({
+                  text: "Remove requirement",
+                  classes: "govuk-button--secondary",
+                  preventDoubleClick: true,
+                  href: "/system/return-requirements/" + sessionId + "/remove/" + requirement.index
+                }) if requirements.length >= 2
+              }
+              }
+            ]
+          }) }}
         </div>
       {% endfor %}
     </div>
   {% endif %}
 
   <form method="post">
     <div class="govuk-body">
-      {% if journey == 'no-returns-required' %}
+      {% if noReturnsRequired %}
```

Comment:
Decisions yours. 

The journey is always one instance of a string as far as i can see. 

I assumed it would be safe to just the the returnsRequired flag. But u figured this must be done for a reason.

Reply:
It is always one instance. So, I think its fine/handy to convert that to the boolean value `returnsRequired` in the presenter and use that in the view. But when it comes to the view we can just do `{% if !returnsRequired %}`.